Content.Server/Repairable/RepairableSystem.cs

```diff
@@ -61,7 +61,7 @@ public async void Repair(EntityUid uid, RepairableComponent component, InteractU
                 return;
 
             // Don't repair if it's a cyborg with no soul
-            if (EntityManager.TryGetComponent(uid, out CyborgComponent? cyborgComponent) && (!EntityManager.TryGetComponent(uid, out MindComponent? mindComponent) || mindComponent.Mind == null)){
+            if (HasComp<CyborgComponent>(uid) && (!EntityManager.TryGetComponent(uid, out MindComponent? mindComponent) || mindComponent.Mind == null)){
```

Comment:
```suggestion
            if (HasComp<CyborgComponent>(uid) && (!EntityManager.TryGetComponent(uid, out MindComponent? mindComponent) || mindComponent.Mind == null))
            {
```
Minor formatting issue.